src/main/java/be/syntra/devshop/DevshopFront/configuration/WebConfig.java

```diff
@@ -34,5 +40,37 @@ RestTemplate restTemplate(RestTemplateBuilder builder) {
                 .interceptors(Collections.singleton(interceptor))
                 .build();
     }
+
+    @Bean
+    public CartDto getCurrentCart() {
+        List<Product> productList = new ArrayList<>();
+        return CartDto.builder()
+                .cartCreationDateTime(LocalDateTime.now())
+                .finalizedCart(false)
+                .activeCart(true)
+                .paidCart(false)
+                .products(productList)
```

Comment:
extractie is ok voor leesbaarheid, maar hier mag de productlist gewoon inline -> .products(new ArrayList<>())